api/app/Http/Middleware/Authenticate.php

```diff
@@ -53,6 +60,14 @@ public function handle(Request $request, Closure $next)
                 "message" => "User with specified hash was not found."
             ], 403);
         }
+    
+        if(($user->getRoleId() !== $this->roles->Admin) && $user->isBlocked())
```

Comment:
Does this state that admin can not be blocked?

Reply:
yes. There is only 1 admin for now